vividus-plugin-web-app/src/main/java/org/vividus/steps/ui/web/ElementSteps.java

```diff
@@ -231,6 +234,57 @@ public void doesElementHaveRightCss(String cssName, StringComparisonRule compari
         });
     }
 
+    /**
+     * Checks that the context <b>element</b> has an expected <b>CSS property</b> value.
+     * <p>
+     * In case CSS property (expected and actual) has equivalent <b>RGB/RGBA</b> values,
+     *  the step converts values for comparison by performing the following steps:
+     *  <ul>
+     *  <li>Remove all <i>white space</i> characters from the value for consistent matching;
+     *  <li>If the <b>alpha channel</b> of RGBA value is <b>equal to 1</b>,
+     *   only the <b>RGB</b> part is used for the comparison (e.g., {@code "rgba(r,g,b,1)"} to {@code "rgb(r,g,b)"}).
+     * </ul>
+     * </p>
+     * @param cssName A name of the <b>CSS property</b> to check (e.g., {@code "background-color"})
+     * @param cssValue An expected value of <b>CSS property</b>
+     */
+    @Then(value = "context element has CSS property `$cssName` with value that is equal to `$cssValue`", priority = 1)
```

Comment:
no new steps should be created, the improvements should be embedded into existing steps